project/AppDependencies.scala

```diff
@@ -15,27 +15,32 @@
  */
 
 import sbt._
-
+import play.core.PlayVersion
 
 object AppDependencies {
 
   val compile: Seq[ModuleID] = PlayCrossCompilation.dependencies(
-    shared = Seq("com.ibm.icu" % "icu4j" % "64.2"),
+    shared = Seq(
+      "com.ibm.icu"           %  "icu4j"        % "68.2",
+      "com.typesafe.play"     %% "play"         % PlayVersion.current
```

Comment:
Interested in this change. Does the `PlayVersion.current` method detect the current version using the version in the Play `sbt-plugin`?

Reply:
Yes exactly. When it is building, depending on which plugin is loaded at that time, it will use the Plugin Version info.